engine/tests/phpunit/Elgg/TestCase.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Elgg;
+
+use Elgg\Di\ServiceProvider;
+use Elgg\Http\Request;
+use ElggDiskFilestore;
+use stdClass;
+use Zend\Mail\Transport\InMemory as InMemoryTransport;
+
+abstract class TestCase extends \PHPUnit_Framework_TestCase {
+
+	/**
+	 * @var Application
+	 */
+	static $_app;
+
+	/**
+	 * @var Config
+	 */
+	static $_config;
+
+	/**
+	 * Constructor
+	 *
+	 * @global stdClass $CONFIG Global config
+	 * @global stdClass $_ELGG  Global vars
+	 */
+	public function __construct($name = NULL, array $data = array(), $dataName = '') {
+
+		date_default_timezone_set('America/Los_Angeles');
+
+		error_reporting(E_ALL | E_STRICT);
+
+		/**
+		 * This is here as a temporary solution only. Instead of adding more global
+		 * state to this file as we migrate tests, try to refactor the code to be
+		 * testable without global state.
+		 */
+		global $CONFIG;
+		$CONFIG = (object) [
```

Comment:
Can we require use a common script to keep these values in sync with the legacy bootstrap or would that defeat the purpose?